server/api.js

```diff
@@ -263,4 +265,41 @@ router.post("/bid/:bidId/status", async (req, res) => {
 	}
 });
 
+router.post("/sign-in", async (req, res) => {
+	const { email, password } = req.body;
+
+	if (!email || !password) {
+		return res.status(400).json({});
+	}
+
+	try {
+		const userResult = await db.query("SELECT * FROM users WHERE email = $1", [
+			email,
+		]);
```

Comment:
```suggestion
                hash  = bcrypt.hashSync(password, SALT);
		const userResult = await db.query("SELECT * FROM users WHERE email = $1 && password_hash = $2", [
			email, hash
		]);
```

Make SALT a global string with some random phrase. If there are no results, return `401`